doc/base/classes.xml

```diff
@@ -604,6 +607,7 @@
 			<argument index="3" name="a8" type="int">
 			</argument>
 			<description>
+			Makes a color from red, gree, blue and alpha.
```

Comment:
Color8 is specifically for constructing a Color from 8-bit values (0-255), in contrast to the Color constructor (0.0-1.0)